.eslintrc

```diff
@@ -32,7 +32,22 @@
     // custom rules
     "@typescript-eslint/lines-between-class-members": "off",
     "@typescript-eslint/no-empty-function": "off",
+    "@typescript-eslint/no-throw-literal": "off",
     "@typescript-eslint/no-explicit-any": "error",
+    // we need to disable the "base" rule to avoid conflicts
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        "args": "all",
+        "argsIgnorePattern": "^_",
+        "caughtErrors": "all",
+        "caughtErrorsIgnorePattern": "^_",
+        "destructuredArrayIgnorePattern": "^_",
+        "varsIgnorePattern": "^_",
+        "ignoreRestSiblings": true
+      }
+    ],
```

Comment:
inspiration for all this: https://johnnyreilly.com/typescript-eslint-no-unused-vars